{
	"id": "RLSA-2021:4161",
	"advisories": [
		{
			"content": {
				"id": "RLSA-2021:4161",
				"title": "Moderate: python-jinja2 security update",
				"description": "The python-jinja2 package contains Jinja2, a template engine written in pure Python. Jinja2 provides a Django inspired non-XML syntax but supports inline expressions and an optional sandboxed environment. \n\nSecurity Fix(es):\n\n* python-jinja2: ReDoS vulnerability due to the sub-pattern (CVE-2020-28493)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Rocky Linux 8.5 Release Notes linked from the References section.",
				"severity": [
					{
						"type": "vendor",
						"source": "errata.rockylinux.org",
						"vendor": "MODERATE"
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28493"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1928707"
					},
					{
						"source": "errata.rockylinux.org",
						"url": "https://errata.rockylinux.org/RLSA-2021:4161"
					}
				],
				"published": "2021-11-09T08:26:43Z"
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-28493",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
							"base_score": 7.5,
							"base_severity": "HIGH",
							"temporal_score": 7.5,
							"temporal_severity": "HIGH",
							"environmental_score": 7.5,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-20->CWE-400"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28493"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "rocky:8",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python-jinja2",
										"architectures": [
											"src"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:2.10.1-3.el8"
											}
										],
										"fixed": [
											"0:2.10.1-3.el8"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-jinja2",
										"architectures": [
											"noarch"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:2.10.1-3.el8"
											}
										],
										"fixed": [
											"0:2.10.1-3.el8"
										]
									}
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "rocky-errata",
		"raws": [
			"vuls-data-raw-rocky-errata/2021/RLSA-2021:4161.json"
		]
	}
}
